packages/components/src/organisms/BannerText/BannerTextContent.tsx

```diff
@@ -21,6 +21,10 @@ export interface BannerTextContentProps extends HTMLAttributes<HTMLDivElement> {
    * The label used at the link.
    */
   linkText: string
+  /**
+   * Link opens in a new tab is true
+   */
+  targetBlank?: boolean
```

Comment:
```suggestion
  linkTargetBlank?: boolean
```

can you pls rename this prop in all occurrences? Thanks! 🥺